packages/authhero/.storybook/HonoJSXWrapper.tsx

```
@@ -0,0 +1,45 @@
+/** @jsxImportSource react */
+import React from "react";
+
+/**
+ * Wrapper component to render Hono JSX components in Storybook
+ * This takes the already-rendered HTML string from a Hono component
+ */
+export function HonoJSXWrapper({ html }: { html: string }) {
+  return React.createElement("div", {
+    dangerouslySetInnerHTML: { __html: html },
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)